docs/how-to/unified_memory.rst

```diff
@@ -0,0 +1,595 @@
+pointer can be used on both host and device. Using pinned memory instead of
+pageable memory on the host can lead an improvement in bandwidth.
+
+While UMM can provide numerous benefits, it is also important
+to be aware of the potential performance overhead associated with UMM.
+Therefore, it is recommended to thoroughly test and profile your code to
+ensure it is indeed the most suitable choice for your specific use case.
+
+System Requirements
+===================
+Unified memory is supported on Linux by all modern AMD GPUs from the Vega
+series onward. Unified memory management can be achieved with managed memory
+allocation and, for the latest GPUs, with a system allocator.
+
+The table below lists the supported allocators. The allocators are described in
+the next chapter.
+
+.. list-table:: Supported Unified Memory Allocators
+    :widths: 40, 25, 25, 25
+    :header-rows: 1
+    :align: center
+
+    * - Architecture
+      - ``hipMallocManaged()``
+      - ``__managed__``
+      - ``malloc()``
+    * - MI200, MI300 Series
+      - ✅
+      - ✅
+      - ✅ :sup:`1`
+    * - MI100
+      - ✅
+      - ✅
+      - ❌
+    * - RDNA (Navi) Series
+      - ✅
+      - ✅
+      - ❌
+    * - GCN5 (Vega) Series
+      - ✅
+      - ✅
+      - ❌
+
+✅: **Supported**
+
+❌: **Unsupported**
+
+:sup:`1` Works only with ``XNACK=1``. First GPU access causes recoverable page-fault.
+
+Unified Memory Programming Models
+=================================
+
+- **HIP Managed Memory Allocation API**:
+The ``hipMallocManaged()`` is a dynamic memory allocator available at all GPUs
+with unified memory support. For more details please visit
+:doc:`reference page:reference/unified_memory_reference`.
+
+- **HIP Managed Variables**:
+The ``__managed__`` declaration specifier, which serves as its counterpart, is
+supported across all modern AMD cards and can be utilized for static
+allocation.
+
+- **System Allocation API**:
+Starting with the MI300 series, it is also possible to reserve unified memory
+via the ``malloc()`` system allocator.
+
+If it is wondered whether the GPU and the environment are capable of supporting
+unified memory management, the ``hipDeviceAttributeConcurrentManagedAccess``
+device attribute can answer it:
+
+.. code-block:: cpp
+
+    #include <hip/hip_runtime.h>
+    #include <iostream>
+
+    int main() {
+        int d;
+        hipGetDevice(&d);
+
+        int is_cma = 0;
+        hipDeviceGetAttribute(&is_cma, hipDeviceAttributeConcurrentManagedAccess, d);
+        std::cout << "HIP Managed Memory: "
+                  << (is_cma == 1 ? "is" : "NOT")
+                  << " supported" << std::endl;
+        return 0;
+    }
+
+Example for Unified Memory Management
+-------------------------------------
+
+The following example shows how to use unified memory management with
+``hipMallocManaged()``, function, with ``__managed__`` attribute for static
+allocation and standard  ``malloc()`` allocation. The Explicit Memory
```

Comment:
Where is EMM presented for comparison? Do you mean in the figure? Consider adding a link to the section, so users don't have to scroll back and forth.

Reply:
Sorry, I meant in the last tab of the tabset. I updated this section.